.github/workflows/major-version-upgrade.yml

```diff
@@ -165,3 +165,13 @@ jobs:
         with:
           name: output-diff.diff
           path: test/JDBC/Info/output-diff.diff
+
+      - name: CI
+      on: [push]
+      jobs:
+        build:
+          runs-on: ubuntu-latest
+          steps:
+          - uses: actions/checkout@v2
+          - name: Setup upterm session
+            uses: lhotari/action-upterm@v1
```

Comment:
Not 100% sure if it would work since we're creating another job and not a step (might create a new environment or may cause an error, not too sure). We would also need an if condition to always run this even if the above failed `if: always()` Try replacing this with this: 

```
- name: Setup upterm session
        if: always()
        uses: lhotari/action-upterm@v1
```
(from lines 51-53 https://github.com/Bit-Quill/babelfish_extensions/blob/da22b8df7c62e18a536aba48c897d053f5345034/.github/workflows/jdbc-tests.yml which is from this commit when I was experimenting with it: https://github.com/babelfish-for-postgresql/babelfish_extensions/pull/811/files/da22b8df7c62e18a536aba48c897d053f5345034)

Reply:
Thank you! Updating it.